src/safe_apps/admin.py

```diff
@@ -23,12 +24,24 @@ def queryset(self, request: Any, queryset: QuerySet[SafeApp]) -> QuerySet[SafeAp
         return queryset
 
 
+class TagInline(admin.TabularInline[Model]):
```

Comment:
Will we be able to create tag by typing right in the Safe App input?

https://djangopackages.org/grids/g/tagging/ – looks like there are additional packages for this purpose, or are they not needed?

Reply:
> Will we be able to create tags by typing right in the Safe App input?

It'd be similar to what we have for `Feature` and `Wallet` for chains. We can iterate on the admin part as we move along with this feature.

> https://djangopackages.org/grids/g/tagging/ – looks like there are additional packages for this purpose, or are they not needed?

I prefer to not introduce 3rd party dependencies just for this. I would start with what Django offers out of the box.